components/track/TrackList.vue

```diff
@@ -12,7 +13,37 @@ withDefaults(
     skeletonCount: 5,
   }
 );
-const { setCurrentSong } = inject(MediaPlaylistInjectionKey)!;
+const { setCurrentSong, addSongToQueue } = inject(MediaPlaylistInjectionKey)!;
+const showDropDownForTrackId = ref(0);
+
+const toggleDropdownForTrack = (trackId: number) => {
+  if (showDropDownForTrackId.value === trackId) {
+    showDropDownForTrackId.value = 0;
+  } else {
+    showDropDownForTrackId.value = trackId;
+  }
+};
+
+const dropdownMenuItemsForTrack = (track: TrackModel): DropdownMenuItem[] => {
+  const items = [];
+
+  if (track?.meta?.parent?.id) {
+    items.push({ text: "Go to album", link: `/album/${track.meta.parent.id}` });
```

Comment:
The link might have to change after my PR #88 is merged. I can update this when either one is merged.